app/services/reports/mis/securable.rb

```diff
@@ -0,0 +1,25 @@
+module Reports
+  module MIS
+    module Securable
+      def sanitise
+        @line.map { |field| sanitise_field(field) }
+      end
+
+      def sanitise_field(field)
+        field.nil? || field.is_a?(Numeric) ? field : prefix_if_necessary(field)
+      end
+
+      def prefix_if_necessary(field)
+        starts_with_special_character?(field.to_s) ? prefix(field.to_s) : field
+      end
+
+      def starts_with_special_character?(str)
+        %w[= + - @ %].include?(str.first)
+      end
+
+      def prefix(field)
+        "'#{field}"
+      end
+    end
+  end
+end
```

Comment:
Would this module be better named `Sanitizable`  to match up with the method name?

Reply:
Good point.  I'll change it

Reply:
@stephenrichards I've changed this, can you re-review at some point please?